assembly/std/index.ts

```diff
@@ -307,7 +325,9 @@ export function derKeys(keysSer: StaticArray<u8>): Array<StaticArray<u8>> {
 /**
  * Converts data to base58.
  *
- * @param data -
+ * @param data - the string data to convert to base58.
+ *
+ * @returns the converted data
  *
  */
 export function toBase58(data: string): string {
```

Comment:
in "return env.toBase58(data);", the toBase58 function is marked as non existant in env